example_router_test.go

```diff
@@ -0,0 +1,25 @@
+package mux_test
```

Comment:
We should move this test case in the file: `example_route_test.go`

Reply:
modev example